packages/slice-machine/src/features/changes/PushChangesButton.tsx

```diff
@@ -0,0 +1,79 @@
+import { Button, Skeleton, Tooltip } from "@prismicio/editor-ui";
+import {
+  type ComponentPropsWithoutRef,
+  type ElementRef,
+  type FC,
+  type ReactNode,
+  Suspense,
+  forwardRef,
+} from "react";
+
+import { useGitIntegrationExperiment } from "@src/features/settings/git/useGitIntegrationExperiment";
+import { useLinkedGitRepos } from "@src/features/settings/git/useLinkedGitRepos";
+import { ErrorBoundary } from "@src/ErrorBoundary";
+
+type PushChangesButtonProps = Required<BarePushChangesButtonProps>;
+
+export const PushChangesButton: FC<PushChangesButtonProps> = (props) => (
+  <ErrorBoundary renderError={renderError}>
+    <Suspense fallback={<PushChangesButtonSkeleton />}>
+      <PushChangesButtonUsingGitIntegrationExperiment {...props} />
+    </Suspense>
+  </ErrorBoundary>
+);
+
+const PushChangesButtonUsingGitIntegrationExperiment: FC<
+  PushChangesButtonProps
+> = (props) => {
+  const gitIntegrationExperiment = useGitIntegrationExperiment();
+  return gitIntegrationExperiment.eligible ? (
+    <PushChangesButtonUsingLinkedGitRepos {...props} />
+  ) : (
+    <BarePushChangesButton {...props} />
+  );
+};
+
+const PushChangesButtonUsingLinkedGitRepos: FC<PushChangesButtonProps> = (
+  props,
+) => {
+  const { linkedGitRepos } = useLinkedGitRepos();
+  return linkedGitRepos.length > 0 ? (
+    <Tooltip
+      content="This repository has been set up to use the Git workflow."
```

Comment:
Is there room to add an extra bit of information?

> This project uses the Git workflow. Push your changes to ${provider} to update Prismic.

`${provider}` would be "GitHub," "GitLab," etc.